mini_pupper_bringup/launch/hardware_interface.launch.py

```diff
@@ -38,6 +39,13 @@ def generate_launch_description():
         description='if the robot has imu sensor'
     )
 
+    port = LaunchConfiguration("lidar_port")
+    port_launch_arg = DeclareLaunchArgument(
```

Comment:
```suggestion
    lidar_port_launch_arg = DeclareLaunchArgument(
```

and put it in `LaunchDescription([ ])` please